github/table_github_repository_content.go

```diff
@@ -0,0 +1,150 @@
+package github
+
+import (
+	"context"
+	"github.com/google/go-github/v48/github"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableGitHubRepositoryContent() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_repository_content",
+		Description: "List the content in a repository (list directory, or get file content",
+		List: &plugin.ListConfig{
+			Hydrate:           tableGitHubRepositoryContentList,
+			ShouldIgnoreError: isNotFoundError([]string{"404"}),
+			KeyColumns: []*plugin.KeyColumn{
```

Comment:
For key columns, are we also able to pass in the `ref`, e.g., commit, branch, that the [API](https://docs.github.com/en/rest/repos/contents#get-repository-content) lists?